src/main/java/ch/njol/skript/expressions/ExprKeyed.java

```diff
@@ -0,0 +1,85 @@
+package ch.njol.skript.expressions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.*;
+import ch.njol.skript.expressions.base.WrapperExpression;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.ExpressionType;
+import ch.njol.skript.lang.KeyProviderExpression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Keyed")
+@Description({
+	"This expression is used to explicitly pass the values of an expression alongside its keys.",
+	"For example, when setting a list variable or passing an expression to a function.",
```

Comment:
```suggestion
	"This expression is used to explicitly pass the keys of an expression alongside its values.",
	"For example, when setting a list variable or passing an expression to a function.",
```
Shouldn't it be the other way around?

Reply:
I'm not sure, my thought process was putting the values first because that's the main thing the user is passing. I feel like if we put the keys first they could think that they're passing a list of the keys (similar to `indices of {_list::*}`). What do you think?

Reply:
I think that the phrasing `explicitly pass the x alongside y` implies that y is the normal thing, and the x is the new thing being focused on, which is why i would put keys first